### PR TITLE
Update FAQ page

### PR DESCRIPTION
**Pull request by marcolussetti**
Should be fixing #97

### CODE REVIEW

**Comment by marcolussetti** [comment]
Blah, not sure why it's showing the remote-tracking merge as a commit given it should not be relevant to this and is already in the master...

File: faq.html [hunk 1]
```diff
@@ -125,19 +126,19 @@ <h3 class="widget-title">Become a member!</h3>
...
                 <!-- Facebook Button -->
                 <div class="fb-follow" data-href="https://www.facebook.com/pages/TRUSU-Computer-Science-Club/343428029131317?fref=ts" data-colorscheme="light" data-layout="button" data-show-faces="true"></div>
                 <!-- Twitter Button -->
-                <a class="twitter-follow-button" href="https://twitter.com/trucsclub" data-show-count="false" data-lang="en">Follow @trucsclub</a>  
+                <a class="twitter-follow-button" href="https://twitter.com/trucsclub" data-show-count="false" data-lang="en">Follow @trucsclub</a>
             </div>
         </aside>
-			</div><!-- /Sidebar -->
-		</div>
+            </div><!-- /Sidebar -->
```
Comment (mitchhentges) [inline faq.html]: While you're cleaning up indentation, this is a little funny.

Comment (marcolussetti) [inline faq.html]: That does look funny, let me take a look.

**Comment by mitchhentges** [comment]
:+1:

**Pull request merged**